### PR TITLE
Full cluster initialize + CKAN instance on minikube

### LINKED ISSUE #68: Running CCO on "minikube"
# Job story

When I use and interact with CKAN Cloud Operator (CCO), I want to be able to run it in local development, so I can debug and develop the codebase and running CKANs in it without standing up a whole cluster on a major cloud provider.

# Context

Currently, we have a range of manual steps that are under- or un-documented, to run CKAN Cloud. And, the assumption is that one is deploying onto a public cloud (GCP, AWS, Azure). This creates friction for both new users (who want to run the thing and see if it addresses their needs), and for DevOps team members already running CCO to manage clusters of CKAN (who need an easier way to standup clusters for testing, debugging, and development purposes).

# Acceptance criteria

- [ ] "Getting stared with CKAN Cloud" documentation that takes me from a terminal, to having CKAN Cloud Operator running on minikube on my local development machine
  - [ ] I can execute basic CCO commands
  - [ ] I can move on to next steps in documentation to deploy a CKAN instance into my setup
- [ ] Any supporting code changes to run a working cluster on minikube

# Note:
"Minikube" is not a strict requirement - any local kubernetes system could work.
We should choose one supported (free) local k8s flavor which is supported on most operating systems, and make sure documentation also includes links to installation instructions for that selected solution.

### PR DESCRIPTION
**Pull request by akariv**
Fixes #68

### CODE REVIEW

File: ckan_cloud_operator/providers/solr/manager.py [hunk 1]
```diff
@@ -137,9 +137,12 @@ def solr_curl(path, required=False, debug=False):
 
 def zk_list_configs():
     pod_name = kubectl.get('pods', '-l', 'app=provider-solr-solrcloud-zk', required=True)['items'][0]['metadata']['name']
-    lines = list(kubectl.check_output(f'exec {pod_name} zkCli.sh ls /configs').decode().splitlines())[5:]
-    assert len(lines) == 1
-    return [name.strip() for name in lines[0][1:-1].split(',')]
+    try:
+        lines = list(kubectl.check_output(f'exec {pod_name} zkCli.sh ls /configs').decode().splitlines())[5:]
+        assert len(lines) == 1
```
Comment (aluminiumgeek) [inline ckan_cloud_operator/providers/solr/manager.py]: Why not just `return []` here then, without `assert`

Comment (akariv) [reply ckan_cloud_operator/providers/solr/manager.py]: Fixed, thanks!

File: ckan_cloud_operator/providers/ckan/deployment/helm/manager.py [hunk 2]
```diff
@@ -287,9 +287,9 @@ def _init_solr(instance_id, dry_run=False):
     solr_status = solr_manager.get_collection_status(instance_id)
     logs.debug_yaml_dump(solr_status)
     if not solr_status['ready']:
-        logs.info('Creating solr collection', collection_name=instance_id, solr_config='ckan_28_default')
+        logs.info('Creating solr collection', collection_name=instance_id, solr_config='ckan_default')
```
Comment (zelima) [inline ckan_cloud_operator/providers/ckan/deployment/helm/manager.py:290]: why `ckan_default` is not it for CKAN 2.6?

File: ckan_cloud_operator/providers/ckan/deployment/helm/manager.py [hunk 3]
```diff
@@ -287,9 +287,9 @@ def _init_solr(instance_id, dry_run=False):
...
         if not dry_run:
-            solr_manager.create_collection(instance_id, 'ckan_28_default')
+            solr_manager.create_collection(instance_id, 'ckan_default')
```
Comment (zelima) [inline ckan_cloud_operator/providers/ckan/deployment/helm/manager.py:292]: why `ckan_default`, is not it for CKAN 2.6?

**Review by zelima — COMMENTED** [review]
A small comment about SOLR schema

File: ckan_cloud_operator/providers/ckan/deployment/helm/manager.py (hunk 3 above)
Comment (akariv) [reply ckan_cloud_operator/providers/ckan/deployment/helm/manager.py:292]: Might be a mistake, feel free to fix

**Pull request merged**